tests/cxx/enums.cc

```diff
@@ -42,6 +42,13 @@ Struct1::IndirectEnum3 ie3;
 // IWYU: UnnamedEnumItem2 is...*enums-i4.h
 auto ie4 = UnnamedEnumItem2;
 
+// No need even in opaque declaration if enumeration or enumerator isn't named
+// explicitly (it already should be elsewhere in the scope). No matter, whether
```

Comment:
"Scope" is a slightly dangerous word, because it has such a precise definition in C and C++. I wonder if we could phrase this differently. Something like `in this case it is already declared earlier in the translation unit`?